### PR TITLE
Add 2.0 API pollers routes

### PR DESCRIPTION
**Pull request by brianparry**
This change includes the following methods/routes.  @srinia6 will be submitting a follow-on PR with unit tests for these routes.

* GET /pollers/library
* GET /pollers/library/{identifier}
* GET /pollers
* GET /pollers/{identifier}
* POST /pollers
* PATCH /pollers/{identifier}
* DELETE /pollers/{identifier}
* PATCH /pollers/{identifier}/pause
* PATCH /pollers/{identifier}/resume
* GET /pollers/{identifier}/data
* GET /pollers/{identifier}/data/current

### CODE REVIEW

**Comment by brianparry** [comment]
Coverage should improve when @srinia6 submits her PR with unit tests.

**Comment by brianparry** [comment]
@RackHD/corecommitters @geoff-reid @srinia6 @dalebremner @abildw

**Comment by jlongever** [comment]
@brianparry Will there be any functional tests created for these new APIs?

**Comment by brianparry** [comment]
@jlongever Yes.  We're planning to start creating functional tests for all of the 2.0 routes soon.  We added some test stories to our backlog on Friday.  I expect we will start working on them next sprint.

File: lib/serializables/v2/pollers.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+// Copyright 2015, EMC, Inc.
```
Comment (WangWinson) [inline lib/serializables/v2/pollers.js]: created on 2015 or 2016 ?

**Comment by dalebremner** [comment]
:+1:

**Comment by yyscamper** [comment]
:+1:

**Comment by srinia6** [comment]
test this please

**Comment by jlongever** [comment]
+1

**Pull request merged**